lib/serializer.ts

```diff
@@ -27,16 +27,22 @@ export interface SerializationOptions {
    * Warns or errors if stability annotations are missing on an export.
    * Supports experimental, stable and deprecated.
    */
-  onStabilityMissing?: string;  // 'warn' | 'error' | 'none'
+  onStabilityMissing?: DiagnosticType;
 }
 
+export enum DiagnosticType {
```

Comment:
why an enum, if you just want the three string values it should be
export type DiagnosticType = 'warn'|'error'|'none';
(and I'd name it DiagnosticSeverity)

Reply:
Ill do the changes for the enum tomorrow, as I dont have access to my
machine today. If feel that it needs to done today. Feel free to update my
PR :)

Reply:
hey @alexeagle managed to do the requested changed today. I just pushed it.

Reply:
Just a quick question, when would you consider using string enums over string union types?